src/renderer/components/settings-general-appearance.tsx

```diff
@@ -193,13 +224,47 @@ export class AppearanceSettings extends React.Component<
             </a>. The easiest way to get started is to clone one of the two existing
             themes and to add your own colors.
           </p>
+          <p>
+            Additionally, if you wish to import a Monaco Editor theme, just pick your JSON file and Fiddle will do the rest.
```

Comment:
```suggestion
            Additionally, if you wish to import a Monaco Editor theme, pick your JSON file and Fiddle will do the rest.
```

I'm a little worried about how optimistic that line is given how likely this feature is to break in the future 🙈 Can we rephrase it to speak about an attempt to import it?